india_compliance/gst_india/overrides/test_journal_entry.py

```
@@ -0,0 +1,95 @@
+import re
+
+import frappe
+from frappe.tests import IntegrationTestCase
+from frappe.utils import getdate
```

Flake8 (7.3.0)

[error] 'frappe.utils.getdate' imported but unused

(F401)